packages/syntax-core/src/TextArea/TextArea.tsx

```diff
@@ -51,6 +51,12 @@ type TextAreaProps = {
    * Value of the TextArea
    */
   value: string;
+  /**
+   * Boolean of whether the TextArea should be resizable
+   *
+   * @default true
+   */
+  resize?: boolean;
```

Comment:
@somethiiing could we change this to take in the values of the CSS resize attribute? That way we avoid the boolean attribute and allow developers to be more specific:
https://developer.mozilla.org/en-US/docs/Web/CSS/resize

```suggestion
  resize?: "none" | "horizontal" | "vertical" | "both";
```

Reply:
@somethiiing one day we can also add something like `auto` to automatically resize the text area once browser support is here: https://chriscoyier.net/2023/09/29/css-solves-auto-expanding-textareas-probably-eventually/

Reply:
awesome! ill keep a an eye out for when enough browser support happens!

Reply:
also, updated with css resize attribute types!